packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -91,6 +91,14 @@ export type ComboBoxProps<T> = {
    * @beta
    */
   enableVirtualization?: boolean;
+  /**
+   * Callback fired when dropdown menu is opened.
+   */
+  onExpand?: () => void;
+  /**
+   * Callback fired when dropdown menu is closed.
+   */
+  onCollapse?: () => void;
```

Comment:
Shouldn't these be `onShow` and `onHide` to be consistent with our other components?

Reply:
I am not sure about naming, whether it will be clear what these props do, ofc they can read JSDocs and then it is obvious.

Reply:
Renamed